cli/config.go

```diff
@@ -0,0 +1,273 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"strconv"
+
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/riotgames/key-conjurer/api/core"
+)
+
+type maybeLegacyID string
```

Comment:
We already mentioned the legacy ID stuff above - We had to have a way to have new keyconjurer understand old keyconjurer configuration, and this ugly hack is it. We attempt to parse old IDs from integers and that is represented with this type.